### ISSUE TITLE
[Spotify Player] ...

### ISSUE THREAD

**Issue by harshBx** [raycast/extensions#9631]
### Extension

https://www.raycast.com/mattisssa/spotify-player

#### Error:
```
Error: Command failed with exit code 1: osascript
407:412: syntax error: Expected end of line, etc. but found identifier. (-2741)

M9:run-applescript.js:38:21

---
35:         timedOut,
36:         command: "osascript",
37:         options,
38:         parentError: new Error(),
39:     })

### Steps To Reproduce

install the spotify app on macos and than install raycast, 
than add raycast spotify extension and than uninstall spotify app from mac 
now try playing the song using extension

### Current Behaviour

_No response_

### Expected Behaviour

_No response_

**Comment by jatindotdev** [comment]
It needs the spotify app to be installed since it tries to open the app when no active client is present

#9511 addresses this issue and adds support for playing with other client like daemon services, etc.

waiting for @peduarte's review

**Comment by peduarte** [comment]
Should be fixed in #9511
